python/lsst/ts/standardscripts/maintel/setup_mtcs.py

```diff
@@ -0,0 +1,380 @@
+        )
+        self.config = None
+        self.mtcs = MTCS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        schema = """
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: https://github.com/lsst-ts/ts_externalscripts/blob/master/python/lsst/ts/externalscripts/maintel/setup_mtcs.py
+        title: SetupMTCS v1
+        description: Configuration for SetupMTCS SAL Script.
+        type: object
+        additionalProperties: false
+        properties:
+          ccw_following:
+            description: >-
+              enable ccw following mode? (default: True)
+            type: boolean
+            default: true
+          overrides:
+            description: Overrides configurations for different components.
+            type: object
+            additionalProperties: false
+            properties:
+              mtptg: 
+                description: Override configuration for MTPtg.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+              mtmount: 
+                description: Override configuration for MTMount.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+              mtrotator: 
+                description: Override configuration for MTRotator.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+              mtm1m3: 
+                description: Override configuration for MTM1M3.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: Default
+              mtm2:
+                description: Override configuration for MTM2.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+              mthexapod_1:
+                description: Override configuration for Camera Hexapod.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+              mthexapod_2:
+                description: Override configuration for M2 Hexapod.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+        """
+        return yaml.safe_load(schema)
+
+    async def configure(self, config):
+        """Handle script input configuration.
+
+        Parameters
+        ----------
+        config: `types.SimpleNamespace`
+          Configuration data. See `get_schema` for information about data
+          structure.
+        """
+        self.log.debug(f"Enable CCW following: {config.ccw_following}")
+        self.config = config
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+        self.activity_estimated_duration = dict(
```

Comment:
define this attribute in the `__init__` method instead...

Reply:
Done